src/pages/Points.js

```diff
@@ -167,49 +159,13 @@ function Points() {
               )}
               <Grid.Row>
                 <Grid.Column>
-                  <Responsive minWidth={700}>
```

Comment:
This shouldn't be deleted. (even though it will be soon 😅). Please pull from master and merge the current changes in the website